core/pkg/server/handler.go

```diff
@@ -866,6 +876,17 @@ func (h *Handler) handleSummary(_ *service.Record, summary *service.SummaryRecor
 	h.summaryHandler.updateSummaryDelta(summaryRecord)
 }
 
+func (h *Handler) handleTbrecord(record *service.Record) {
+	if !h.tbHandler.Active {
```

Comment:
what do you think about hiding active in the handler and just have it part of start.
Also save it in the logs in case multiple starts are done?

Reply:
actually i think this patter is better. let's keep this way